javaslang-test/src/main/java/javaslang/test/Arbitrary.java

```diff
@@ -197,6 +199,18 @@
         return size -> random -> Gen.choose(-size, size).apply(random);
     }
 
+    static Arbitrary<LocalDateTime> localDateTime() {
+        return localDateTime(ChronoUnit.DAYS);
+    }
+
+    static Arbitrary<LocalDateTime> localDateTime(ChronoUnit unit) {
+        return localDateTime(LocalDateTime.now(), unit);
+    }
+
+    static Arbitrary<LocalDateTime> localDateTime(LocalDateTime median, ChronoUnit unit) {
+        return size -> random -> Gen.choose(median.minus(size, unit), median.plus(size, unit)).apply(random);
```

Comment:
We move the logic from Gen to Arbitrary like this:

```java
Objects.requireNonNull(median, "median is null");
Objects.requireNonNull(unit, "unit is null");
return size -> {
    final LocalDateTime start = median.minus(size, unit);
    final LocalDateTime end = median.plus(size, unit);
    final long duration = Duration.between(start, end).toMillis();
    final Gen<Long> from = Gen.choose(0, duration);
    return random -> start.plus(from.apply(random), ChronoUnit.MILLIS);
}
```

_(Note: We name the parameters directly in the error messages)_

Some Gen.localDateTime tests might be obsolete or need to be moved to ArbitraryTest...